File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/12468_:::_Zapping/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int a, b;
		for(String line, aux[]; !(line=br.readLine()).equals("-1 -1");)
		{
			aux=line.split(" ");
			a=Integer.parseInt(aux[0]);
			b=Integer.parseInt(aux[1]);
			System.out.println(Math.min(Math.abs(a-b), Math.min(a,b)+100-Math.max(a,b)));
		}
		//-----------------------------------------------------------
		br.close();
	}
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/12468_:::_Zapping/Main.c
#include "stdio.h"
#include "stdlib.h"

#define MIN(a,b) (((a)<(b))?(a):(b))
#define MAX(a,b) (((a)>(b))?(a):(b))

int main()
{
	int a, b;

	while(scanf("%d %d", &a, &b), (a!=-1||b!=-1))
		printf("%d\n", MIN(abs(a-b), MIN(a,b)+100-MAX(a,b)));

	return 0;
}
